docs/colour.colorimetry.rst

```diff
@@ -53,6 +53,7 @@ Spectral Data Generation
     sd_blackbody
     sd_constant
     sd_ones
+    sd_rayleigh_jeans
```

Comment:
the `rayleigh_jeans_law` definition should be documented in the section just after.